darts/tests/test_timeseries.py

```diff
@@ -764,6 +764,20 @@ def test_with_values(self):
         # should not fail if nr samples is not the same:
         series.with_values(np.random.rand(5, 10, 2))
 
+    def test_cumsum(self):
+        cumsum1 = TimeSeries.from_dataframe(self.series1.pd_dataframe().cumsum())
+        # cumsum on cumsum2 should be equal to n=2 cumsum on series 1
+        cumsum2 = TimeSeries.from_dataframe(cumsum1.pd_dataframe().cumsum())
+        assert self.series1.cumsum() == cumsum1
+        assert self.series1.cumsum(n=2) == cumsum2
+
+        with pytest.raises(ValueError):
+            self.series1.diff(n=0)
+        with pytest.raises(ValueError):
+            self.series1.diff(n=-5)
+        with pytest.raises(ValueError):
+            self.series1.diff(n=0.2)
```

Comment:
This tests diff rather then cumsum. From my comment below I would opt for dropping `n` support